src/Compilers/CSharp/Test/Semantic/Semantics/PatternSwitchTests.cs

```diff
@@ -1733,9 +1728,9 @@ static void Main(string[] args)
                 // (8,18): error CS0150: A constant value is expected
                 //             case new object() is int x1:
                 Diagnostic(ErrorCode.ERR_ConstantExpected, "new object() is int x1").WithLocation(8, 18),
-                // (9,17): warning CS0162: Unreachable code detected
+                // (9,42): error CS0165: Use of unassigned local variable 'x1'
```

Comment:
>(9,42): error CS0165: Use of unassigned local variable 'x1' [](start = 19, length = 59)

This error seems misleading since `x1` would be assigned if this line was reached. If this is a cascading error, can it be removed?

Reply:
Reported as [https://github.com/dotnet/roslyn/issues/9154](https://github.com/dotnet/roslyn/issues/9154).

---
In reply to: [176465734](https://github.com/dotnet/roslyn/pull/25544#discussion_r176465734) [](ancestors = 176465734)